src/swift/Sources/Models/TypeModel.swift

```diff
@@ -88,6 +99,8 @@ struct TypeModel {
             self.init(from: src)
         case let src as DictionaryType:
             self.init(from: src)
+        case let src as MetatypeType:
+            self.init(from: src.referenceType)
```

Comment:
Can you give an example of a `MetatypeType`

Reply:
Nope, it's just some crazy thing that swift-ast popped out of nowhere, so I added it. I think it was in the context of generics.

Reply:
Lol ok sounds good to me